services/harmony/app/backends/workflow-orchestration/work-item-updates.ts

```diff
@@ -564,9 +564,13 @@ export async function preprocessWorkItem(
     durationMs = new Date().getTime() - resultStartTime;
     logger.debug('timing.HWIUWJI.getResultItemSize.end', { durationMs });
   } catch (e) {
-    errorMessage = 'Could not get result item file size';
+    errorMessage = 'Could not get result item file size from output STAC';
     logger.error(errorMessage);
-    logger.error(e);
+    logger.error('Caught exception:', {
+      message: e.message,
```

Comment:
With JSON logging and cloud metrics they can't have a field called message so it will likely break parsing or get converted to something else. Is there something that was missing with the call to just `logger.error(e)`? I thought that should be showing the stack trace.

Reply:
Maybe `Caught exception: ${e.message}\`, {name: e.name, stack: e.stack}`

Reply:
You just raised a great point and pointed out the real cause of this ticket. There is no exception logged in Cloud Metrics is not because the exception is not logged by Harmony. It is actually caused by Cloud Metrics not able to handle the logging with message!